src/components/products.vue

```diff
@@ -1,34 +1,22 @@
 <template>
-  <div class="home">
-    <v-container class="teal lighten" fluid>
-      <v-row dense>
-        <v-col class="d-flex justify-space-between">
-          <div v-if="isError">
-            <v-alert color="red" dismissible type="success">{{
-              error
-            }}</v-alert>
-          </div>
-          <v-card
-            v-else
-            class="pa-3"
-            outlined
-            v-for="product in products"
-            :key="product.id"
-          >
-            <v-img height="200" contain :src="product.img"></v-img>
+  <v-timeline>
+    <v-timeline-item v-for="product in products" :key="product.id">
+      <div v-if="isError">
```

Comment:
Comme on a dit l'erreur ne doit pas être dans la boucle..
Parce que soit on a nos produits soit on a une erreur..
Regarde même en haut, tu trouve qu'on avait la div avec isError et le "else" dans la v-card qui contient le v-for..

Reply:
ok